### DISCUSSION TITLE
Is there any option to convert .rd5 file to .xodr

### DISCUSSION THREAD

**Discussion by ajay1606** [IPG-Automotive-UK/Aslan-CarMaker#8] [General]
Hello, 

Would it be possible to convert the road file format (.rd5) to OpenDrive (.xodr) format using CM Gui? 

Thanks in advance.

Regards,
Ajay

**Comment by bogotm** [comment]
Hello,

Yes it's possible. There are some limitations described in the CarMaker documentation, but exporting to OpenDRIVE 1.6 should be possible in CarMaker 11. Previous version can also export to earlier OD versions.

In Scenario Editor, there is an "Export" button (4th from the left) in the main toolbar. Scenario Editor can be reached from the main GUI in Parameters - Scenario/Road.

**Reply by ajay1606** [reply]
@Thanks for your quick response and appreciated it. 

Due to this [issue](https://github.com/IPG-Automotive-UK/Aslan-CarMaker/discussions/6#discussioncomment-3404248) i have rollback my CM version to 10.2.1. Anyhow I will try CM11 again.

But currently in CM 10.2.1, Am able to convert OpenDrive (.xodr ) to RaodFile (.rd5), but unable to do reverse RaodFile (.rd5) to OpenDrive (.xodr ) .


Export WIndow:

In this case, I have uploaded file in .rd5 file format and trying to export it into .xodr format. But there unable to save those file.
![image](https://user-images.githubusercontent.com/29722843/184837367-bcc399d9-6552-4263-a69a-a09beecd8719.png)

![image](https://user-images.githubusercontent.com/29722843/184838060-987b2d11-abc3-409a-bc43-17bb37b40ead.png)


Regards,
Ajay

**Reply by bogotm** [reply]
Hello,
I guess you are using some KML files directly and referencing them in the scenario. CarMaker cannot take a reference to a KML file and export it to an OpenDRIVE since both are not really native CarMaker formats. You will have to find a way to define the road entirely in ScenarioEditor without referencing external files.

**Reply by ajay1606** [reply]
@bogotm Thanks for your kind response.
